### ISSUE TITLE
Update Mumble before Debian 11 freeze

### ISSUE THREAD

**Issue by Neustradamus** [mumble-voip/mumble#4673]
Hello Mumble team,

Not possible to release a new build before the Debian 11 freeze (2021-01-12)?
Next Debian will be here in 2023.

It is possible to update Mumble?
- https://salsa.debian.org/pkg-voip-team/mumble
- https://tracker.debian.org/pkg/mumble

Note: There is already a 1.3.3:
- https://github.com/mumble-voip/mumble/releases
cc @crknadle

Thanks in advance.

**Comment by Krzmbrzl** [comment]
Unfortunately it will not be possible to release 1.4.0 within that time frame. It's just not ready yet.
And there is currently no new release.in the 1.3.x series planned either.

**Comment by Neustradamus** [comment]
@Krzmbrzl: Thanks but it will be nice to have the 1.3.3 in Debian before the freeze.

**Comment by Krzmbrzl** [comment]
Well this is not in our hands as we do not maintain the Debian (nor any other Linux) package ourselves. If 1.3.3 is not included in Debian yet, you'll have to report that to the respective maintainer :point_up: 
I actually don't know who that is though...

**Comment by TredwellGit** [comment]
https://tracker.debian.org/pkg/mumble

**Comment by Neustradamus** [comment]
@Krzmbrzl, @TredwellGit; Yes, it is @crknadle.
- https://github.com/mumble-voip/mumble/issues/4673#issue-778533318

**Comment by Krzmbrzl** [comment]
Okay in that case I guess you can either wait on him responding to this issue (as you've pinged him already) or write an email :+1:

**Comment by crknadle** [comment]
I am the maintainer of the Mumble package for Debian.
@Neustradamus already contacted me via email, but when trying to reply the email is being rejected because of a problem with one or more of the email addresses in the recipient list (or so the error message says **shrug**).

I plan on updating the Debian package to Mumble 1.3.3 before the "soft freeze" on Feb 12th. (There was concern that Debian 11 was freezing on Jan 12th, but that's for packages requiring transitions (i.e. libraries) and for build essentials.)

https://release.debian.org/bullseye/freeze_policy.html

**Comment by crknadle** [comment]
Mumble in Debian has been updated to 1.3.3. It should transition to Debian Testing in about 5 days, unless a serious bug is reported on the package (which I doubt). There's a small chance I might do one more upload of v 1.3.3 before the Debian 11 freeze only because I see that there are some Lintian errors on overrides that no longer apply.

**Comment by Neustradamus** [comment]
Thanks @crknadle :)
